assets/src/application/ui/layout/confirmDelete/ConfirmDelete.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Popconfirm } from 'antd';
+import { PopconfirmProps } from 'antd/lib/popconfirm';
+import { __ } from '@wordpress/i18n';
+
+const ConfirmDelete: React.FC<Partial<PopconfirmProps>> = ({ children, ...props }) => {
+	return (
+		<Popconfirm title={__('Are you sure delete it?')} okText={__('Yes')} cancelText={__('No')} {...props}>
+			{children}
+		</Popconfirm>
```

Comment:
```suggestion
const ConfirmDelete: React.FC<Partial<PopconfirmProps>> = ({ children, message, ...props }) => {
const title = message ? message : __('Are you sure you want to delete this?');
	return (
		<Popconfirm title={title } okText={__('Yes')} cancelText={__('No')} {...props}>
			{children}
		</Popconfirm>
```

Reply:
`
const title = message ? message : __('Are you sure you want to delete this?');
`
nice extraction
 